### ISSUE TITLE
loot at mb storage areas

### ISSUE THREAD

**Issue by M-ace** [NCC-Lykos/neocron-classic#124]
can the loot be looked at for the storage areas in mb?  a friend and i started tanks in cm and we are broke as shit lol.  the mobs drop nothing of value.  most of the stuff yo's wont even take.  and the stuff that we can sell we only get 1 or 2 nc for lol.  if we were leveling in nc city we can make a small fortune selling bones ect.  as it is, i cant even buy armor to go outside to fight launchers and i have 30 str now.

**Comment by ghost** [comment]
We'll look into it when we get to our Military Base overhaul.  However, it's worth noting that when you create your runners, it does mention that City Mercs, The Fallen Angels, and Twilight Guardian are not for beginners.  

The issue that you're having is one of those reasons.

**Comment by M-ace** [comment]
hmmm, so they intended for u to be broke.  too bad neither of us have hack skill.  wonder what the cyberguards and sentenals drop.  I ran out to the bunker to try my hand at a launcher cuz i know they drop better loot.  died in 4 rockets.  wish i could afford the armor for my level lol.  I did die alot being a melee tank and all tho, thats why i didnt buy my first weapon until i had 24 str.  Death toll is a mofo lol.  Dem sentinals was whippin my arse!  anywho, i reported it, up to yall to decide weather u wanna do something about it. 👯
